docs/maya/ui_overview.md

```diff
@@ -81,6 +85,22 @@ In Nodes section, there is a menu item to create an AdnEdgeEvaluator node.
 
 #### Deformers
 
+In the Deformers section, there is a menu item to open the UI to create an AdnSkinMerge node.
```

Comment:
Should we generalize the message and not focus on the `AdnSkinMerge`?
```suggestion
In the Deformers section, there is a menu item to open AdnSkinMerge node creation UI.
```

Reply:
Done